### DISCUSSION TITLE
Any tips to consider to keep high performance and low resource usage?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [google/model-viewer#4432] [Q&A] [answered]
Hello, I'm sorry if my question might sound too generic. I am integrating a GBL model into my webpage, and obviously this implies more RAM usage and some initial lag during page load. Apart from that, I'm fairly satisfied with the final result. 

Generally speaking, are there any recommendations to follow in order to keep the loading smooth and, mostly, to keep the usage of RAM as low as possible? I'm referring to:

- Setting to check when exporting the GBL file
- Configuration parameters for the `<model-viewer>` tag

Thanks

**Comment by elalish** [comment, marked as answer]
Yes, I'd recommending reading this [FAQ](https://modelviewer.dev/docs/faq.html#entrydocs-technical-questions-lighthouse) and the related link. On the export side, your best bet for lowering RAM usage is to use [KTX2](https://modelviewer.dev/examples/loading/#ktx2Support) compression on your textures. A good way to achieve this is with the [gltf-transform](https://www.npmjs.com/package/@gltf-transform/cli) utility.
